services/memory/.env.example

```
@@ -18,3 +18,8 @@ IBEX_RANK_WEIGHT_FREQUENCY=0.05
 # Embedder HTTP client (PR-C)
 IBEX_MEMORY_EMBEDDING_BASE_URL=http://127.0.0.1:8004
 # IBEX_EMBEDDING_API_TOKEN=
+
+# PII stage (m3.C.1 / ADR-0054) — separate from injection quarantine threshold
+IBEX_MEMORY_PII_REDACT_MIN_CONFIDENCE=0.70
+IBEX_MEMORY_PII_SPACY_MODEL=en_core_web_md
+IBEX_MEMORY_MAX_CONTENT_CHARS=10000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The IBEX_MEMORY_MAX_CONTENT_CHARS key should go before the IBEX_MEMORY_PII_REDACT_MIN_CONFIDENCE key

(UnorderedKey)